### PR TITLE
Try to clean up the installing-and-updating-drivers flow

### PR DESCRIPTION
**Pull request by Erisa**

### CODE REVIEW

**Review by FloatingMilkshake — APPROVED** [review]

File: src/wiki/installing-and-updating-drivers.md [hunk 1]
```diff
@@ -181,46 +181,54 @@ Safe mode is a diagnostic tool, which loads Windows with only the essential driv
...
-   ![The Windows Recovery Environment's troubleshoot screen.](./img/installing-and-updating-drivers/winre/troubleshoot_screen.png)
+      ![The Windows Recovery Environment's troubleshoot screen.](./img/installing-and-updating-drivers/winre/troubleshoot_screen.png)
 
-3. Finally, click on "System Restore" and follow the instructions
+   3. Click on "System Restore" and follow the instructions
+
+      ![Screen in the Windows Recovery Environment, titled "Advanced options", with the option "System Restore" highlighted.](./img/installing-and-updating-drivers/winre/advanced_options_screen.png)
+
+      
+   4.
```
Comment (FloatingMilkshake) [inline src/wiki/installing-and-updating-drivers.md:217]:
```suggestion
```
Guessing the 4 was added by mistake

**Review by FloatingMilkshake — CHANGES_REQUESTED** [review]

**Review by FloatingMilkshake — APPROVED** [review]

**Comment by julsehn** [comment]
wait a sec

**Review by julsehn — CHANGES_REQUESTED** [review]
Page direction wrong

**Comment by itsWindows11** [comment]
What's wrong with the page "direction"?

**Comment by julsehn** [comment]
> What's wrong with the page "direction"?

Page direction at the last line in the file.

**Pull request merged**